lib/inline_svg/action_view/helpers.rb

```diff
@@ -12,7 +12,7 @@ def inline_svg(filename, transform_params={})
             configured_asset_file.named filename
           end
         rescue InlineSvg::AssetFile::FileNotFound
-          return "<svg><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
+          return "<svg class='svg-not-found'><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
```

Comment:
I wonder if this CSS class should be something more specific to avoid potential collisions with existing user-defined classes. What do you think?